Assets/UGSSamples/Plugins/PlayerProfile/SampleAuthenticator.cs

```diff
@@ -0,0 +1,45 @@
+
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Unity.Services.Authentication;
+using Unity.Services.Core;
+using UnityEngine;
+
+namespace Unity.Services.Samples
+{
+    /// <summary>
+    /// The Drag & Drop Samples shared implementation of the Unity Authentication Service
+    /// </summary>
+    public static class SampleAuthenticator
```

Comment:
Should this be called, UnityServiceAuthenticator instead? Not making exclusive to sample

Reply:
I could be down with that.